src/components/shared/Entity/Actions/Savebutton.tsx

```diff
@@ -71,16 +76,18 @@ function EntityCreateSaveButton({ disabled, formId, onFailure }: Props) {
 
                 showNotification({
                     description: intl.formatMessage({
-                        id: `${messagePrefix}.captureCreate.createNotification.desc`,
+                        id: `${messagePrefix}.createNotification.desc`,
                     }),
                     severity: 'success',
                     title: intl.formatMessage({
-                        id: `${messagePrefix}.captureCreate.createNotification.title`,
+                        id: `${messagePrefix}.createNotification.title`,
                     }),
                 });
             },
             () => {
-                onFailure(`${messagePrefix}.save.failedErrorTitle`);
+                onFailure({
+                    error: { title: `${messagePrefix}.save.failedErrorTitle` },
+                });
```

Comment:
Needed to pass the error object and not just the title string. Cauased the errors not to show up on the entity creation